docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '2'
+volumes:
+    prometheus_data: {}
+    grafana_data: {}
+
+services:
+  prometheus:
+    image: prom/prometheus
+    container_name: prometheus
+    volumes:
+      - ./compose/prometheus/prometheus.yml:/etc/prometheus/prometheus.yml
+      - prometheus_data:/prometheus
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+      - '--storage.tsdb.path=/prometheus'
+    expose:
+      - 9090
+    links:
+      - fastly-exporter:fastly-exporter
+    ports:
+      - 9090:9090
+  
+  grafana:
+    image: grafana/grafana
+    depends_on:
+      - prometheus
+    ports:
+      - 3000:3000
+    volumes:
+      - ./compose/grafana/provisioning/:/etc/grafana/provisioning/
+      - grafana_data:/var/lib/grafana
+
+  fastly-exporter:
+    image: mrnetops/fastly-exporter
+    command: -token ${FASTLY_API_TOKEN}
```

Comment:
You want to leverage the upcoming env var to avoid sticking the token in `ps` output?

Reply:
I'll adjust it to pure ENV when @ljagiello finishes their edits and we merge.  I suspect he'll be done before we finish out the review.

I know you're dot a docker (or I assume docker-compose) guy but you should check this out, it's hella cool to see in action.

Reply:
@mrnetops it's done now.

Reply:
Docker hub updated (@peterbourgon we probably want to tag and release after this PR) and update added!

Nice addition and suggestion @ljagiello @peterbourgon